public/friendlycode/js/fc/filesystem-sync.js

```diff
@@ -0,0 +1,104 @@
+define(["jquery"], function($) {
+  "use strict";
+
+  var FileSystemSync = {};
+
+  function pushFileChange(url, csrfToken, fs, path) {
+    var options = {
+      contentType: "application/json",
+      headers: {
+        "X-Csrf-Token": csrfToken
+      },
+      type: "PUT",
+      url: url
+    };
+
+    function send() {
+      var request = $.ajax(options);
+      request.done(function() {
+        if(request.readyState !== 4) {
+          return;
+        }
+
+        if(request.status !== 201 && request.status !== 200) {
+          // TODO: handle error case here
+          console.error("Server did not persist file");
+          return;
+        }
+
+        console.log("Successfully persisted ", path);
```

Comment:
I understand why you want this for debugging, but let's not do this.  What is our plan around events on `bramble` for success/error cases with syncing file changes to the server?  We could do something like:

```js
bramble.trigger("syncFinished", path); // when it works
...
bramble.trigger("syncFailed", path, reason); // when it fails
```

And then clients can do whatever with that info.